README.md

```diff
@@ -122,25 +152,37 @@ Given the following DOM tree
 </div>
 ```
 
-we could find all of the elements with `className === 'banner'` by calling `document.getElementsByClassName('banner')`.
+we could find all of the elements with `className === 'banner'` by calling
+`document.getElementsByClassName('banner')`.
 
 **Try it out!**
 
-Inspect the web page again, this time making note of a className. Get all elements with that className and give 'em a look. Remember, you can use a `for` loop to loop through them. (You can also assign the return value of `document.getElementsByClassName()` to a variable: `var elements = document.getElementsByClassName('banner')`.)
+Inspect the web page again, this time making note of a className. Get all
+elements with that className and give 'em a look. Remember, you can use a `for`
+loop to loop through them. (You can also assign the return value of
+`document.getElementsByClassName()` to a variable: `var elements =
+document.getElementsByClassName('banner')`.)
 
 #### `document.getElementsByTagName()`
 
-Suppose you don't know an element's ID but you do know its tag name (the tag name is the main thing between the `<>`, e.g., `'div'`, `'span'`, `'h1'`, etc.). Since tag names aren't unique, this method returns an HTMLCollection of 0 to many nodes with the given tag.
+Suppose you don't know an element's ID but you do know its tag name (the tag
+name is the main thing between the `<>`, e.g., `'div'`, `'span'`, `'h1'`, etc.).
```

Comment:
can we add an `<article>` or a `<header>` too. try to sell the value of the sem elements